src/opera/pge/disp_s1/schema/algor_param_disp_s1_schema.yaml

```diff
@@ -39,15 +39,11 @@ runconfig:
             init_method: str()
         output_options:
             # Output (x, y) resolution (in units of input data).
-            output_resolution:
-                x: int(required=False)
-                y: int(required=False)
+            output_resolution: any(include('x_y_object'), null())
             # Alternative to specifying output resolution: Specify the (x, y) strides (decimation
             #   factor) to perform while processing input. For example, strides of [4, 2] would turn an
             #   input resolution of [5, 10] into an output resolution of [20, 20].
-            strides:
-                x: int(required=False)
-                y: int(required=False)
+            strides: any(include('x_y_object'), null())
             # Options for `create_dataset` with h5py.
             hdf5_creation_options:
                 chunks: list()
```

Comment:
Just double checking is this without parameters because it is a single list and 'indexes' is a list of lists?

Reply:
I believe that is correct. Indexes is a list of lists.  There are some comments that hint at what the correct schema construct should be but some of this is a best-guess at this point.